mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -581,6 +581,21 @@
     <!-- Onboarding toolbar selection card label for 'bottom' option, used by Nimbus experiments. -->
     <string name="onboarding_customize_toolbar_bottom_option">Bottom</string>
 
+    <!-- Onboarding header for the theme selection card, used by Nimbus experiments. -->
+    <string name="onboarding_customize_theme_title" tools:ignore="UnusedResources">Pick a theme</string>
+    <!-- Onboarding sub header for theme selection card, used by Nimbus experiments. -->
+    <string name="onboarding_customize_theme_description" tools:ignore="UnusedResources">See the web in the best light.</string>
+    <!-- Onboarding theme selection card label for 'save and continue' button, used by Nimbus experiments. -->
+    <string name="onboarding_customize_theme_save_and_continue_button">Save and start browsing</string>
+    <!-- Onboarding theme selection card label for 'skip' button, used by Nimbus experiments. -->
+    <string name="onboarding_customize_theme_skip_button">Skip</string>
+    <!-- Onboarding theme selection card label for 'dark' option, used by Nimbus experiments. -->
+    <string name="onboarding_customize_theme_dark_option">Dark</string>
+    <!-- Onboarding theme selection card label for 'light' option, used by Nimbus experiments. -->
+    <string name="onboarding_customize_theme_light_option">Light</string>
+    <!-- Onboarding theme selection card label for 'system default' option, used by Nimbus experiments. -->
```

Comment:
The comment here doesn't seem to apply to the string, since it doesn't describe what "System auto" is or does 🤔